wkcuber/compress.py

```diff
@@ -164,4 +167,6 @@ def compress_mags(
 if __name__ == "__main__":
     args = create_parser().parse_args()
     setup_logging(args)
-    compress_mags(args.source_path, args.layer_name, args.target_path, args.mag, args)
+    compress_mags(
+        Path(args.source_path), args.layer_name, Path(args.target_path), args.mag, args
```

Comment:
I believe argparse supports `type=Path` so we may be able to go without these conversions. Not sure if that creates incompabitilies with other parts of the wkcuber, though

Reply:
Yes that works